velox/vector/VariantToVector.cpp

```diff
@@ -37,10 +37,19 @@ ArrayVectorPtr variantArrayToVectorImpl(
   BufferPtr nulls =
       AlignedBuffer::allocate<bool>(variantArraySize, pool, bits::kNullByte);
 
+  TypePtr type;
+  constexpr bool isDecimalType =
+      (KIND == facebook::velox::TypeKind::SHORT_DECIMAL ||
+       KIND == facebook::velox::TypeKind::LONG_DECIMAL);
```

Comment:
Use this method in the if block,
```
inline bool isDecimalKind(TypeKind typeKind) {
  return (
      typeKind == TypeKind::SHORT_DECIMAL ||
      typeKind == TypeKind::LONG_DECIMAL);
}

```

Reply:
Thanks for the suggestion, the compile time check is not possible with this method.